.claude/skills/layer5-blog-writer/scripts/check_post.py

```
@@ -0,0 +1,275 @@
+import mesh_palette as palette
+
+SKILL_ROOT = Path(__file__).resolve().parent.parent
+TAXONOMY = SKILL_ROOT / "references" / "tags-categories.md"
+
+RASTER_SUFFIXES = (".jpg", ".jpeg", ".png", ".webp")
+REQUIRED_FIELDS = [
+    "title", "date", "author", "thumbnail", "darkthumbnail",
+    "description", "type", "category", "tags", "featured", "published",
+]
+DATE_RE = re.compile(r"^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} [+-]\d{4}$")
+DASH_RE = re.compile(r"[–—]")
```

Ruff (0.16.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)